Features/LootableContainers.cs

```diff
@@ -45,7 +45,12 @@ public override PointOfInterest[] RefreshData()
 			if (rootItem is not { IsContainer: true })
 				continue;
 
-			if (rootItem.TemplateId != KnownTemplateIds.BuriedBarrelCache && rootItem.TemplateId != KnownTemplateIds.GroundCache)
+			if (rootItem.TemplateId != KnownTemplateIds.BuriedBarrelCache 
+					&& rootItem.TemplateId != KnownTemplateIds.GroundCache 
+					&& rootItem.TemplateId != KnownTemplateIds.air_drop_common
+					&& rootItem.TemplateId != KnownTemplateIds.air_drop_medical
+					&& rootItem.TemplateId != KnownTemplateIds.air_drop_supply
+					&& rootItem.TemplateId != KnownTemplateIds.air_drop_weapon)
 				continue;
```

Comment:
```suggestion
            if (ExcludedContainerIds.Contains(rootItem.TemplateId))
                continue;
```

I'd recommend redoing this with something like a HashSet since we are checking for a larger list of possible containers. This HashSet should be a static near line 15 so we don't build it on every loop.

Reply:
Done.